stories/Console.stories.tsx

```diff
@@ -1093,3 +1093,106 @@ export const ImageFiles = () => {
     </div>
   );
 };
+
+export const Working = () => {
```

Comment:
Whats this story?

Reply:
it's a "working" version - i made it to test the input stuff, but decided it might be useful to have something that functions even as state variables